tensorflow/python/tools/saved_model_cli.py

```diff
@@ -137,24 +146,121 @@ def _show_inputs_outputs(saved_model_dir, tag_set, signature_def_key, indent=0):
       meta_graph_def, signature_def_key)
 
   indent_str = '  ' * indent
+
   def in_print(s):
     print(indent_str + s)
 
   in_print('The given SavedModel SignatureDef contains the following input(s):')
   for input_key, input_tensor in sorted(inputs_tensor_info.items()):
     in_print('  inputs[\'%s\'] tensor_info:' % input_key)
-    _print_tensor_info(input_tensor, indent+1)
+    _print_tensor_info(input_tensor, indent + 1)
 
   in_print('The given SavedModel SignatureDef contains the following '
            'output(s):')
   for output_key, output_tensor in sorted(outputs_tensor_info.items()):
     in_print('  outputs[\'%s\'] tensor_info:' % output_key)
-    _print_tensor_info(output_tensor, indent+1)
+    _print_tensor_info(output_tensor, indent + 1)
 
   in_print('Method name is: %s' %
            meta_graph_def.signature_def[signature_def_key].method_name)
 
 
+def _show_defined_functions(saved_model_dir, indent=0):
```

Comment:
Please remove the indent parameter. We can just use a constant value unless there is need for adding this.